definitions/variable/emissions/tag_land-removal-options.yaml

```diff
@@ -0,0 +1,16 @@
+- Land Removal Option:
+    - Agroforestry:
+        description: agroforestry
+    - Afforestation:
+        description: afforestation
+    - Biochar:
+        description: conversion of biomass to recalcitrant carbon via pyrolysis and its
+          subsequent application to croplands to increase soil carbon stocks
+    - Soil Carbon Management:
+        description: soil carbon management techniques
```

Comment:
This would be useful to expand. Even a basic description like "enhancing the carbon content of soils through ..."

Reply:
Added the first part of your sentence.